bot/cogs/git.py

```diff
@@ -131,6 +131,24 @@ async def init(self, ctx):
             )
         )
         await ctx.send(msg)
+    @git.command(help'Prints info on the git pull command')
+    async def pull(self, ctx):
+        msg = ManMessage(
+            name='git-pull'
+            synopsis='git pull git pull [options] [<repository> [<refspec>...]]',
+            options=[
+                ('-q, --quiet'),('-v, --verbose'),
+                ('--[no-]recurse-submodules[=yes|on-demand|no]', 'This option controls if new commits of all populated submodules should be fetched and updated')
+            ],
+            short_desc='Incorporates changes from a remote repository into the current branch.',
+            long_desc=(
+                'Incorporates changes from a remote repository into the current branch.In its default mode, git pull is shorthand for git fetch followed by git merge FETCH_HEAD.'
+                '\n'
+                'More precisely, \'git pull\' runs git fetch with the given parameters and calls git merge to merge the retrieved branch heads into the current branch. With --rebase, it runs git rebase instead of git merge.'
```

Comment:
I use a tuple-like syntax to combine multiple strings into a longer string. They are automatically concatenated, without any delimiter. Consider splitting up longer strings into multiple lines.
**Note**: since no delimiter is added, you have to add a space to the end of the string.

Additionally, if you want to start a new line, end your line with '\n'. If you want a blank line, use a '\n' on both the line above to get a new line, and then another one on the empty line. Like this:
```python
('This is a longer paragraph that will have to be split '
 'up into multiple lines.\n'
 '\n'
 'The next paragraph is separated by an empty line')
```
which becomes:
```
This is a longer paragraph that will have to be split up into multiple lines.

The next paragraph is separated by an empty line.
```

Reply:
I'll keep that in mind